mammoth/tests/test_beam_search.py

```diff
@@ -316,6 +385,7 @@ def test_beam_is_done_when_n_best_beams_eos_using_min_length(self):
                 beam.update_finished()
                 self.assertTrue(beam.done)
 
+    @unittest.skip('attention no longer returned')
```

Comment:
you do have a keyword about returning attn in the model though?

Reply:
Yes, x-transformers is able to return the attention.
However, it is returned in a different format from the previous implementation, and the use cases for it have been removed. I decided to remove the return instead of reimplementing it.

Maybe the test should be removed as well. Unless someone feels like reimplementing this feature, and fixing the test.